backend/doc/setting.md

```diff
@@ -14,10 +14,10 @@ The priority order is (from less to more priority):
 - configuration file
 - environnement variables
 
+<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}|{config_name: <68}|"  with all app enabled to update this list properly --->
```

Comment:
```suggestion
<!--- Maintainer: use tracimcli dev parameters list --template "| {env_var_name: <74}| {config_file_name: <63}|{config_name: <68}|"  with all apps enabled to update this list properly --->
```

Why put an important information in an HTML comment that will not be rendered?

Reply:
The problem here is : 
- Document how to update the markdown file
- Distinct clearly what is backside setup (maintaining the list to have up to date documentation) and the documentation.
If you have a better suggestion, i'm listening.

Reply:
if you are not maintaining the documentation there is no new to see this.

Reply:
This distinction must not be done by hiding things on the rendered document I think. If you need to split things, maybe use a different section in the file or a different file?

Reply:
If you put this anywhere else, someone will not see it and will forgot it.

Reply:
Okay, but if you hide it in a comment, people looking at the rendered document will not see it. I can't manage to follow you here. @grignards?

Reply:
note: there is a documentation about the command line use here, but it's more generic, this comment is really about maintaining properly the list just here in this documentation. The command here is already documented somewhere else.

Reply:
Oh, I see. This comment is intented at people editing this file. Still, they could very well think of editing the document before opening it.
I don't know what is best. Another opinion would not be too much I think.

Reply:
I do like the idea of putting the instructions in the document which is better than having no precise instructions of how to update it. Even better would be to change the `dev parameters list` command in order to integrate this "standard" template directly as an option/command, but this is not part of this issue.